<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bitcart Vault - Pending permission</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QLabel" name="warning_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>741</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;An external website {website} is requesting access to your hardware wallet.&lt;br/&gt;Approve only if you trust the source.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;
color: white;</string>
   </property>
   <property name="text">
    <string>Approve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reject_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: red;
color: white</string>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
